modules/investigating-kernel-crashes.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-operating-system-issues.adoc
+
+[id="investigating-kernel-crashes"]
+= Investigating kernel crashes
+
+== Enabling kdump
+
+`kdump` (included in `kexec-tools`) is a service providing a crash dumping mechanism. The service enables you to save the contents of the system’s memory for later analysis.
```

Comment:
Generally we avoid parenthetical statements because of the IBM style guide, so I would suggest rewriting this to something along the lines of: 

`kdump`, which is included in `kexec-tools`, is a service providing a crash dumping mechanism. You can use `kdump` to save the contents of the system’s memory for later analysis.

Reply:
Also avoid starting the sentence with a lowercase word, avoid the possessive if possible, and hyphenate the adjectival phrase "crash-dumping", so something like:

> The `kdump` service, included in `kexec-tools`, provides a crash-dumping mechanism. You can use this service to save system memory content for later analysis.